packages/auth/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "@standardnotes/auth",
+  "version": "1.0.0",
+  "engines": {
+    "node": ">=14.0.0 <16.0.0"
+  },
+  "description": "Auth SDK for Standard Notes projects",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "files": [
+    "dist"
+  ],
+  "publishConfig": {
+    "access": "public"
+  },
+  "author": "Karol Sójko <karolsojko@gmail.com>",
```

Comment:
Personal email?

Reply:
hmm I assumed it has to be the one which I use as a github account

Reply:
will update in that case